airbyte-integrations/connectors/destination-snowflake/Dockerfile

```diff
@@ -20,5 +20,5 @@ RUN tar xf ${APPLICATION}.tar --strip-components=1
 
 ENV ENABLE_SENTRY true
 
-LABEL io.airbyte.version=0.4.45
+LABEL io.airbyte.version=0.4.46
```

Comment:
this will change after this PR is merged: https://github.com/airbytehq/airbyte/pull/20631

Reply:
yup, thanks. I'd bumped the version and after pushing noticed your comment to wait for another PR. It was 1 minute earlier :)